components/organisms/login/login.js

```diff
@@ -34,6 +34,7 @@ export const Login = ({
   blockSubmitButton,
   forgotPassword,
   heading,
+  noTitle,
```

Comment:
Change this to `showTitle` with a `true` defaultValue. The condition should also be changed to `{showTitle && </>} `